prolog/comment.pl

```diff
@@ -30,8 +28,9 @@
                            
 comment(S,E) -->
     {
-        tr(S,SS),
-        tr(E,EE)
+        copy_term([S,E],[S2,E2]),
```

Comment:
Why is copy term needed?

Reply:
yep, I needed it for the adaptive comment but I changed it  and now it is removed